pydantic/_internal/_std_types_schema.py

```diff
@@ -69,33 +70,28 @@ def to_enum(__input_value: Any, _: core_schema.ValidationInfo) -> Enum:
         except ValueError:
             raise PydanticCustomError('enum', 'Input is not a valid enum member')
 
-    enum_ref = f'{getattr(enum_type, "__module__", None)}.{enum_type.__qualname__}:{id(enum_type)}'
+    enum_ref = get_type_ref(enum_type)
     literal_schema = core_schema.literal_schema(
         [m.value for m in enum_type.__members__.values()],
-        ref=enum_ref,
     )
-    js_metadata = JsonSchemaMetadata(
-        core_schema_override=literal_schema.copy(),
-        source_class=enum_type,
-        title=enum_type.__name__,
-        description=inspect.cleandoc(enum_type.__doc__ or 'An enumeration.'),
+    updates = {'title': enum_type.__name__, 'description': inspect.cleandoc(enum_type.__doc__ or 'An enumeration.')}
```

Comment:
"An enumeration." seems like perhaps the most confusing description imaginable for a set of choices in JSON schema - should we just remove?

Reply:
I'll see if I can see any reason why it was added where it was added. I don't see much reason to default to having a description, we don't do that for models. If there's nothing obvious I think we can remove it.

Reply:
Investigated — this is actually set as the docstring of the enum via `enum.EnumMeta.__new__`. I think the closest thing we could do is just explicitly look for this string and if it's exactly equal to this, remove it. I'm inclined not to change it for now though, since this has been the behavior all along.